templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Are you sure you want to delete the car {{ object.name }}?</h1>
```

Comment:
The confirmation message incorrectly refers to a car. It should be updated to refer to a manufacturer, such as 'Are you sure you want to delete the manufacturer {{ object.name }}?'.